queries/vue/indents.scm

```diff
@@ -0,0 +1,12 @@
+[
```

Comment:
could also `; inherits: html_tags` like in `vue/highlights.scm` instead of copying the content of `html_tags/indents.scm`

Reply:
There aren't indents queries for html, https://github.com/nvim-treesitter/nvim-treesitter/tree/master/queries/html_tags. @cmizzi you should copy these queries to `html_tags/indents.scm` and then add the `; inherits: html_tags` comment here.

Reply:
There are html indents, but they are in `html` and not in `html_tags`. I could just moved to html tags

Reply:
I moved the `html/indents.scm` to `html_tags` and implemented a `; inherits: html_tags` in `html` and `vue`.